Backend/app/routes/notification.py

```
@@ -0,0 +1,210 @@
+):
+    await db.execute(
+        delete(Notification)
+        .where(Notification.user_id == user["id"])
+        .where(Notification.id.in_(ids))
+    )
+    await db.commit()
+    return
+
+@router.patch("/mark-read", status_code=status.HTTP_200_OK)
+async def mark_notifications_read(
+    ids: List[str] = Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)